CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ All notable changes to this project will be documented in this file.
 ## Unreleased
 ### Added
 ### Changed
+- Allow explicit setting of postgresql DSN
```

Comment:
I guess the `Initial release message is a bit misleading, but this would be the format for changelog entries.

```suggestion
- Allow explicit setting of postgresql DSN from [mcfearsome](https://github.com/mcfearsome)
```